### ISSUE TITLE
suggestions: Minor UI adjustments

### ISSUE THREAD

**Issue by mnotforme** [etesync/etesync-web#156]
So I came from Reddit, and was told to open issues here for them.
**Search Tasks:** the search button requires 2 clicks (1 on the magnifying glass symbol + 1 in the search field). having it always expanded would be better. I currently sometimes confuse it with the NewTask field box.

**indicators for Subtasks:** for subtasks, similar to those used Reddit comments or [something similar](https://i.postimg.cc/wMT4XM5V/msedge-L6pf-Jm1j-ZU.png)

![lines](https://user-images.githubusercontent.com/69506895/89886302-14b2ca80-dbd5-11ea-8f5e-46d4784a6444.png)

**Collapsing subtasks:** Subtasks take a bit too much space and collapsing them allows for greater control

**after exiting a task or subtask, it would be helpful to have it highlighted in the list.

**Mark as Completed:** to easily fix misclicks, waiting for about 5 seconds before hiding a task that is marked as "completed" or adding an "undo" toast.

**Comment by tasn** [comment]
Thanks a lot for all of the suggestions. Could you please elaborate on the search tasks issue? Specifically, what are you suggesting instead of what's currently there?

Thanks again!

**Comment by mnotforme** [comment]
![Tasks Search](https://user-images.githubusercontent.com/69506895/89973500-25148500-dc69-11ea-9be9-140e4a779bd5.png)
im just suggesting that the search bar is expanded as you see in step 2, to reduce the number of clicks needed to search + larger clickable ara for something that is used often.  
but if you personally find it annoying to be always expanded, at least you could have clicking the magnifying glass icon directly enable typing instead of having to click the magnifying glass then clicking the field.

also sometimes I  confuse the newtask field box with search.

**Comment by tasn** [comment]
![Screenshot_2020-08-12 EteSync - Secure Data Sync](https://user-images.githubusercontent.com/108670/89980936-92311600-dc7b-11ea-8c1c-b62699a912a8.png)

This is how it looks here, do you not get the guide text in the "quick add" field? I tried keeping the searchbar always open, but it looks weird.

**Comment by mnotforme** [comment]
you are correct, i do get the indicators, but im just annoyed for having to click twice instead of 1. it would be alright if clicking the magnifying glass symbol would immediately activate the search box.

**Comment by tasn** [comment]
Done.
